src/interfaces/AdventOfCode.d.ts

```diff
@@ -0,0 +1,19 @@
+interface AocLeaderBoard {
+	event: string;
+	owner_id: string;
+	members: { [key: string]: AocMember };
+}
+
+interface AocMember {
+	global_score: number;
+	local_score: number;
+	completion_day_level: { [key: string]: { [key: string]: AocCompletionDayLevel } };
+	stars: number;
+	last_star_ts: number | string;
+	id: string;
+	name: null | string;
+}
+
+interface AocCompletionDayLevel {
+	get_star_ts: string;
+}
```

Comment:
For consistency, can this just be a regular `.ts` file with the interfaces exported, please. Would also be great if the interfaces were prefixed `AOC` rather than `Aoc`.